aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -116,6 +134,25 @@ TORCH_META_FUNC(argmin)
   check_argmax_argmin(*this, "argmin", self, dim, keepdim);
 }
 
+TORCH_META_FUNC2(sum, dim_IntList)
+(const Tensor& self, IntArrayRef dim, bool keepdim, optional<ScalarType> opt_dtype) {
+  ScalarType dtype;
+  const auto& result = maybe_get_output();
+
+  if (result.defined()) {
+    dtype = opt_dtype.value_or(result.scalar_type());
+  } else {
+    dtype = at::native::get_dtype_from_self(self, opt_dtype, true);
+  }
+
+  DimVector dims(dim);
+  maybe_wrap_dims(dims, self.dim());
```

Comment:
I expected something corresponding to this code to be deleted in the other part of this PR, wondering why we can't delete it yet. Is this because this is another composite-y kernel? Kind of looks like it...

Reply:
Not sure if I got your question. Are you asking what's the relation between this PR and the `mean` one? If so, I guess this is my bad. After having ported them both, I realized it didn't make sense to group them into helper functions (they have the same logic, but don't necessarily have to).